browse/templates/abs/extra_services.html

```diff
@@ -52,20 +52,16 @@
   <li><a href="{{url_for('dissemination.pdf', arxiv_id=requested_id)}}" aria-describedby="download-button-info" accesskey="f" class="abs-button download-pdf">Download PDF</a></li>
       {%- endif -%}
       {%- if format == 'latexml' -%}
-  <li>
-    <a href="{{ latexml_url }}" class="abs-button" id="latexml-download-link">HTML (experimental)</a>
-  </li>
+  <li><a href="{{ latexml_url }}" class="abs-button" id="latexml-download-link">HTML (experimental)</a></li>
       {%- endif -%}
       {%- if format == 'src' and current-%}
-  <li><a href="{{url_for('src.e_print', arxiv_id=requested_id)}}" class="abs-button download-eprint">TeX Source</a></li>
+  <li><a href="{{url_for('src.src', arxiv_id=requested_id)}}" class="abs-button download-eprint">TeX Source</a></li>
```

Comment:
Do not change to src unless there is a Jira ticket that explicitly requests this.